src/App.js

```diff
@@ -0,0 +1,59 @@
+import { useState, useEffect } from "react";
+import Cell from "./components/Cell";
```

Comment:
I believe you followed the guide and default export was used there but using named exports (the syntax that we use in the lessons) are more beneficial https://blog.neufund.org/why-we-have-banned-default-exports-and-you-should-do-the-same-d51fdc2cf2ad